lib/engine-jit/Cargo.toml

```diff
@@ -11,16 +11,16 @@ readme = "README.md"
 edition = "2018"
 
 [dependencies]
-wasmer-types = { path = "../types", version = "1.0.2" }
-wasmer-compiler = { path = "../compiler", version = "1.0.2", features = ["translator"] }
-wasmer-vm = { path = "../vm", version = "1.0.2" }
+wasmer-types = { path = "../types", version = "1.0.2", features = ["enable-rkyv"] }
+wasmer-compiler = { path = "../compiler", version = "1.0.2", features = ["translator", "enable-rkyv"] }
+wasmer-vm = { path = "../vm", version = "1.0.2", features = ["enable-rkyv"] }
```

Comment:
If `rkyv` is optional, shouldn't we add an optional feature here that enables `enable-rkyv` on `wasmer-types`, `wasmer-compiler` and `wasmer-vm`? Something like:

```toml
[features]
enable-rkyv = ["wasmer-types/enable-rkyv", …]
```

Reply:
In the case of engine jit, we actually need rkyv for serializing/deserializing (same way, serde was not optional but mandatory).

Reply:
:+1: